lib/process-services-cloud/src/lib/start-task-cloud/components/people-cloud/people-cloud.component.ts

```diff
@@ -82,29 +110,86 @@ export class PeopleCloudComponent implements OnInit {
     }
 
     private async loadUsers() {
-        const roles = [PeopleCloudComponent.ROLE_ACTIVITI_ADMIN, PeopleCloudComponent.ROLE_ACTIVITI_MODELER, PeopleCloudComponent.ROLE_ACTIVITI_USER];
+        const roles = this.getApplicableRoles();
```

Comment:
private async loadUsers() {
        if (!this.hasRoles())
        this.roles = this.getDefaultRoles();
....}

hasRoles() { this.roles ? this.roles.length > 0 }

getDefaultRoles() { return [PeopleCloudComponent.ROLE_ACTIVITI_ADMIN, PeopleCloudComponent.ROLE_ACTIVITI_USER]
( we don't need modeler)